shell.nix

```diff
@@ -3,14 +3,17 @@
   sha256 = "sha256:1k5nvn2yzw370cqsfh62lncsgydq2qkbjrx34cprzf0k6b93v7ch";
 }) {} }:
 
+
 pkgs.mkShell {
   name = "nim-waku-build-shell";
+  stdenv = pkgs.clangStdenv;
 
   # Versions dependent on nixpkgs commit. Update manually.
   buildInputs = with pkgs; [
     git   # 2.37.3
     which # 2.21
     gcc   # 11.3.0
     rustc # 1.63.0
+    clang
```

Comment:
Clang is already included by default in `stdenv` for Darwing. No need to include it explicitly.

Reply:
Please drop clang.

Reply:
Oh, and you don't need the `pkgs.` prefix for `darwin.apple_sdk.frameworks.Security` because of `with pkgs;` above.

Reply:
Addressed in [21f2635](https://github.com/status-im/nwaku/pull/1190/commits/21f263591d0735a6e901978054ba3e2132ebf088)